pandas/core/arrays/period.py

```diff
@@ -765,10 +779,15 @@ def _add_delta(self, other):
         """
         if not isinstance(self.freq, Tick):
             # We cannot add timedelta-like to non-tick PeriodArray
-            raise IncompatibleFrequency("Input has different freq from "
-                                        "{cls}(freq={freqstr})"
-                                        .format(cls=type(self).__name__,
-                                                freqstr=self.freqstr))
+            if np.ndim(other) > 0:
+                # FIXME: What should this be rendered as?
+                other_freq = "FIXME"
```

Comment:
Does "None" make sense here?

Reply:
Either that or something resembling a least common denominator.  None sounds easier.